docs/guides/1230-persistent-cache.md

```diff
@@ -0,0 +1,219 @@
+
+```shell
+# Down buildkit daemon
+docker container stop dagger-buildkitd && docker container rm dagger-buildkitd && docker volume rm dagger-buildkitd
+
+# Import cache on rebuild
+dagger do build --cache-to type=local,mode=max,dest=storage --cache-from type=local,src=storage
+[✔] actions.build.container                                                 2.3s
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.4s
+```
+
+:::info
+In this part, we have how to keep cache in a local filesystem, if you want
+to see more options on local export, look at [Buildkit cache documentation](https://github.com/moby/buildkit/blob/v0.10.1/README.md#local-directory-1)
+:::
+
+## Persistent cache in a remote registry
+
+Buildkit can also import/export cache to a registry. This is a great way to
+share cache between your team and avoid flooding your filesystem.
+
+:::tip
+Using a registry as cache storage is more efficient than local storage because Buildkit will only re-export
+missing layers on multiple runs.
+:::
+
+Let's first deploy a simple registry in your localhost
+
+```shell
+docker run -d -p 6000:5000 --restart=always --name cache-registry registry:2
+```
+
+Then it's not much different from local export
+
+```shell
+dagger do build --cache-to type=registry,mode=max,ref=localhost:5000/cache --cache-from type=registry,ref=localhost:5000/cache
+[✔] actions.build.container                                                 1.3s
+[✔] client.filesystem.".".read                                              0.0s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.1s
+```
+
+:::info
+See more options on registry export at [Buildkit cache documentation](https://github.com/moby/buildkit/blob/v0.10.1/README.md#registry-push-image-and-cache-separately)
+:::
+
+## Persistent cache in GitHub Actions
+
+Buildkit has a great integration to store cache from GitHub Action.  
+That features is really powerful with Dagger because you can cache everything that has not changes in your PR.
+
+It's not much different as `local` or `registry` exports, let's integrate cache in a
+simple workflow
+
+```yaml title=".github/workflows/build-example.cue"
+name: "Dagger export export"
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    # Set cache export environment
+    env:
+      DAGGER_CACHE_FROM: type=gha,scope=dagger-cache-example
+      DAGGER_CACHE_TO: type=gha,mode=max,scope=dagger-cache-example
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+          
+      # Required to retrieve GitHub token    
+      - name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v1
```

Comment:
Not required. See https://github.com/dagger/dagger-for-github/pull/56#issuecomment-1111730526

Reply:
Fixed in https://github.com/dagger/dagger/pull/2035/commits/1651d359cfc25c6d69b505ae6103eeb90a06dc66